src/resources/elements/primeDesignSystem/pinput-numeric/pinput-numeric.html

```diff
@@ -10,8 +10,10 @@
     autocomplete.to-view="autocomplete"
   ></numeric-input>
 
-  <i if.bind="validationState === 'warning'" class="feedback-icon fas fa-exclamation-circle"></i>
-  <i if.bind="validationState === 'error'" class="feedback-icon fas fa-exclamation-triangle"></i>
-  <i if.bind="validationState === 'validating'" class="feedback-icon fas fa-circle-notch loading"></i>
+<template if.bind="!disabled">
```

Comment:
Would the validation stuff be better located in pform-input?

Reply:
I tried that before and I tried it right now as well. It cannot be done because the icons have position: absolute and it won't look centered when you add more elements inside the pform-input "slot". Take this for example:
![image](https://user-images.githubusercontent.com/4083652/154419747-a0d04599-3fec-4062-ba02-ebf932a36210.png)

The icons will appear in the right hand side of the component 😢